config/identify_system.go

```diff
@@ -6,12 +6,14 @@ import (
 )
 
 // Figure out if we're self-hosted or on RDS, as well as what ID we can use - Heroku is treated separately
-func identifySystem(config ServerConfig) (systemType string, systemScope string, systemScopeFallback string, systemID string) {
+func identifySystem(config ServerConfig) (systemID string, systemType string, systemScope string, systemIDFallback string, systemTypeFallback string, systemScopeFallback string) {
 	// Allow overrides from config or env variables
+	systemID = config.SystemID
 	systemType = config.SystemType
 	systemScope = config.SystemScope
+	systemIDFallback = config.SystemIDFallback
+	systemTypeFallback = config.SystemTypeFallback
 	systemScopeFallback = config.SystemScopeFallback
-	systemID = config.SystemID
```

Comment:
I ended up rearranging this since we are not always consistent about how we order these. In most places we have (id, type, scope), which makes sense to me, so that's what I tried to standardize on. I think it also makes sense to keep the fallback values together, rather than intersperse them with the main identity values. In some places (mostly protobuf definitions), we don't have the flexibility to do that, but most cases can be consistent.